### ISSUE TITLE
Splash page redesign

### ISSUE THREAD

**Issue by dudesahn** [yearn/yearn-finance#108]
Includes built with section, blocks alchemy agreement

Add Alchemy (get html snippet from x48)
Add Blocknative

**Comment by dudesahn** [comment]
- Two options here
- - Infomercial style overview page
- - - Add section for “built with”
- - - Splash page needs to show useful information that explains simply how the product works
- - - Should probably include some nice images
- - - No more than 4 pages total (this is flexible but use this as guidelines)
- - - - Splash
- - - - Main overview pane (what does yearn do)
- - - - Partners
- - - - Built with
- - - - Footer (social, links etc)
- - - For any metrics visible on splash we need to make sure the metrics are available before committing
- - - - If metrics are not available the overview page is blocked
- - - - Figure out what is needed to unblock and make bugs
- - - Need mobile designs for overview page
- - Simple splash (zapper style)
- - - When wallet is not connected show splash
- - - When wallet is connected show main site when user visits yearn.finance (/vaults)… zapper does /dashboard
- - - Allow user to “log out” to go back to the splash page
- - - Splash page shows very basic information on one page with a button to connect wallet (see zapper)
- - - Splash page links to documentation, faq, partners
- - - - Intentions here are:
- - - - - Splash page does not get in the way for users who have aleady logged in
- - - - - - This is a major annoyance for sushi/uniswap/compound
- - - - - - Imagine if zapper made you click “enter” every time you viewe their site
- - - - - New users are exposed to useful basic information

**Comment by dudesahn** [comment]
Out of scope for v2, will be included in v3
